### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Anton20HD**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+
+<section>
+</div>
```
Comment (jonathanjonathan001) [inline README.md:3]: Why is there a closing < / div > there?

**Review by jonathanjonathan001 — CHANGES_REQUESTED** [review]
seems to be an unnecessary < / div > on line 3

File: README.md (hunk 1 above)
Comment (Anton20HD) [reply README.md]: Tested with a different workflow badge but didnt seem relevant to this project. Forgot to delete!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+<section>
+<div align="center">
+    <a href="https://github.com/jonathanjonathan001/springbootlab/issues">
+        <img src="https://img.shields.io/github/issues-raw/jonathanjonathan001/springbootlab" alt="Open issues workflow badge"/>
+    </a>
+    <a href="https://github.com/jonathanjonathan001/springbootlab/pulls">
+        <img src="https://img.shields.io/github/issues-pr/jonathanjonathan001/springbootlab" alt="Pull request workflow badge"/>
+    </a>
+    <a href="https://github.com/jonathanjonathan001/springbootlab/issues?q=is%3Aissue+is%3Aclosed">
+        <img src="https://img.shields.io/github/issues-closed-raw/jonathanjonathan001/springbootlab" alt="Open issues workflow badge"/>
```
Comment (jonathanjonathan001) [inline README.md:11]: nit-picking here :) , but on line 11, alt-text should say Closed issues workflow badge, right?

**Review by jonathanjonathan001 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Anton20HD) [reply README.md]: Oh, thats right. Fixed it now

**Review by jonathanjonathan001 — APPROVED** [review]
Lookin' good!

**Review by ArmanAlemirad — APPROVED** [review]
Good job!

**Pull request merged**